prov/gni/src/gnix_rma.c

```diff
@@ -881,6 +1082,8 @@ ssize_t _gnix_rma(struct gnix_fid_ep *ep, enum gnix_fab_req_type fr_type,
 	int rc;
 	int rdma;
 	struct fid_mr *auto_mr = NULL;
+	struct gnix_fab_req *more_req;
+	struct slist_entry *sle;
 
```

Comment:
In that case, I assume I shouldn't put GNIX_FATAL in the other places you mentioned?